### PR TITLE
new libretro cores: higan-sfc and higan-sfc-balanced

### PR DESCRIPTION
**Pull request by ToKe79**
based on #464 request. builds OK for all projects, images building right now. tests will follow.

### CODE REVIEW

**Comment by ToKe79** [comment]
I expect a merge conflict with #474 and #476 because of LIBRETRO_CORES.

**Comment by ToKe79** [comment]
I thought that the `higan` can emulate many consoles, besides Super Famicom ([source](https://higan.readthedocs.io/en/stable/))

> It also includes some level of support for these consoles:
> - Satellaview add-on for the Super Famicom
> - Nintendo Famicom/Nintendo Entertainment System
> - Nintendo Game Boy
> - Nintendo Game Boy Color
> - Sega Master System
> - Sega Game Gear
> - Sega Mega Drive/Genesis
> - NEC PC Engine/TurboGrafx 16 (but not the CD-ROM² System/TurboGrafx-CD)
> - NEC SuperGrafx
> - Bandai WonderSwan
> - Bandai WonderSwan Color

But if it can only emulate SFC, I will change the name (please, confirm). I will have to rebase so or so later :-)

**Comment by ToKe79** [comment]
Oh, and link to images: http://nightly.builds.lakka.tv/special_builds/pr477/

**Comment by RobLoach** [comment]
Higan can emulate all those consoles. There's only been a libretro core put together for `higan_sfc_libretro` so far:
https://gitlab.com/higan/higan/tree/libretro/higan/target-libretro

See [`higan_sfc_libretro.info`](https://github.com/libretro/libretro-super/blob/master/dist/info/higan_sfc_libretro.info) for the matching .info file.

Don't worry about rebasing, it's possible to [squash the commits when merging](https://blog.github.com/2016-04-01-squash-your-commits/#whats-changing) in the GitHub UI.

**Comment by kivutar** [comment]
I confirm that we need to match this file https://github.com/libretro/libretro-super/blob/master/dist/info/higan_sfc_libretro.info

**Comment by RobLoach** [comment]
> link to images: http://nightly.builds.lakka.tv/special_builds/pr477/

Just tested on the RPi2, and it runs! Albeit very slowly... Nicely done!

**Comment by ToKe79** [comment]
The core/lib file is `higan_sfc_libretro.so`, so I will only rename the package to `higan-sfc`. I will wait for the merges of #474 and #476.

**Comment by ToKe79** [comment]
the higan-sfc core is an accuracate core, so I have also added the balanced fork - nSide / higan-sfc-balanced.

File: distributions/Lakka/options [hunk 1]
```diff
@@ -253,7 +253,7 @@
...
+  LIBRETRO_CORES="2048 4do 81 atari800 beetle-bsnes beetle-lynx beetle-ngp beetle-pce beetle-pcfx beetle-psx beetle-saturn beetle-supergrafx beetle-vb beetle-wswan bluemsx bsnes bsnes-mercury cannonball cap32 chailove citra crocods desmume dinothawr dolphin dosbox easyrpg fbalpha fceumm freeintv fuse-libretro gambatte genesis-plus-gx gearboy gme gpsp gw-libretro handy hatari higan-sfc higan-sfc-balanced lutro mame2003 mame2003-plus mame2003-midway melonds meowpc98 mgba mrboom mupen64plus nestopia nxengine o2em openlara parallel-n64 pcsx_rearmed picodrive pocketcdg ppsspp prboom prosystem puae px68k redream reicast sameboy scummvm snes9x snes9x2002 snes9x2005 snes9x2005_plus snes9x2010 stella tgbdual tyrquake uae4arm uzem vbam vecx vice virtualjaguar xrick yabause"
```
Comment (RobLoach) [inline distributions/Lakka/options]: Restored `citra`?

Comment (ToKe79) [reply distributions/Lakka/options]: yes, see #474 :-)

Comment (ToKe79) [reply distributions/Lakka/options]: hmm... but I don't understand why citra is here as a change, when cannonball is not. strange rebase.

Comment (ToKe79) [reply distributions/Lakka/options]: my mistake when merging #476 - don't know how that could happen. But citra should stay enabled.

**Pull request merged**